runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -1730,6 +1730,8 @@ private static String longBossName(String boss)
 			case "hs5":
 			case "hs 5":
 				return "Hallowed Sepulchre Floor 5";
+			case "zalc":
```

Comment:
This should have its own comment header as well as empty lines surrounding it like other sections. eg.

```java
// Zalcano
case "zalc":
	return "Zalcano";
```